controllers/podchaos/types.go

```diff
@@ -84,6 +87,18 @@ func (r *Reconciler) schedulePodChaos(podchaos *v1alpha1.PodChaos, req ctrl.Requ
 		return reconciler.Reconcile(req)
 	case v1alpha1.PodFailureAction:
 		tr = podfailure.NewTwoPhaseReconciler(r.Client, r.Log.WithValues("action", "pod-kill"), req)
+	case v1alpha1.ContainerKillAction:
```

Comment:
why duplicate v1alpha1.ContainerKillAction

Reply:
I don't quite understand the problem. Would you please describe it again?